pom.xml

```diff
@@ -165,25 +165,32 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-            <version>2.9.1</version>
-	    <configuration>
+            <version>3.0.0</version>
+            <configuration>
               <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Please add a comment indicating that this is for building maven-javadoc-plugin 2.x on Fedora 27.
Also, try adding `<doclint>none</doclint>` here (instead of requiring `-Ddoclint=none` on the command line).